src/tab-group.tab.ts

```diff
@@ -56,16 +76,24 @@ export default class Tab extends LitElement {
    */
   @property({ type: Boolean, reflect: true })
   get selected(): boolean {
-    return this.#isSelected;
+    return this.#selected;
   }
 
   set selected(isSelected: boolean) {
-    const hasChanged = isSelected !== this.#isSelected;
-    this.#isSelected = isSelected;
+    const hasChanged = isSelected !== this.#selected;
+    this.#selected = isSelected;
 
-    if (hasChanged) {
+    if (hasChanged && isSelected) {
+      this.dispatchEvent(
+        new Event('selected', {
+          bubbles: true,
+          composed: true,
+        }),
+      );
+    } else if (hasChanged) {
       this.dispatchEvent(
-        new Event('private-selected', {
+        new Event('deselected', {
+          // No need to be composed, as consumers won't use this event directly.
```

Comment:
Matches [Option](https://github.com/CrowdStrike/glide-core/blob/main/src/option.ts#L175-L194).

The `selected` event here is composed, as consumers want to know when a tab is selected. Rather than commenting there, I decided to comment on why `deselected` is **not** composed. Let me know if that's odd.